shuffle/cli/Cargo.toml

```diff
@@ -57,3 +60,11 @@ path = "src/main.rs"
 
 [dev-dependencies]
 tempfile = "3.2.0"
+
+[[test]]
+name = "shuffle-forge-integration-tests"
+harness = false
+
+[[test]]
+name = "shuffle-forge-cli-tests"
```

Comment:
Rather than rename `shuffle-forge-integration-tests` leave it at the old name and just name this one `shuffle-cli-tests` under `/shuffle/cli-tests`